homeworks/franktoth/6_testing/test_bankaccount.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from bankaccount import BankAccount
+
+
+# Fixture to create BankAccounts with different initial balances
+@pytest.fixture(scope="class")
```

Comment:
miért váltasz class scope-ra? a fixture alapból function scope-ra van állítva, ami azt jelenti hogy minden teszt hívás az eredeti fixture állapotot fogja megkapni. Így ha módosít az objektumokon egy teszt, a következő a módosult verziót fogja megkapni. A saját dolgod nehezíted ezzel, mert követni kell hogy melyik teszt hogyan változtatja a balance-t.
Én ezt visszatenném function scope-ra, és magához a tesztekhez sem használnék class-t, csak simán a function-öket írnám meg ahogy órán is tanultuk.

Reply:
Ez egy rossz tanács volt egy Python-os oldalon. Én sem igazán értettem, minek. Okés kivéve.